packages/website/src/pages/index.tsx

```diff
@@ -57,9 +57,9 @@ const FoundationsPages: React.FC = () => (
                 thumbnail="iconography"
             />
             <Tile
-                title="Headings"
-                description="A heading is a title at the top of a page or section. Its distinctive font helps visualize the hierarchy of information."
-                href="foundations/Headings"
+                title="Typekit"
+                description="The Typekit covers the entire spectrum of typography styles designed specifically to work in harmony with the Plasma ecosystem."
```

Comment:
@jlevesquecoveo I improvised something with the description here, feel free to change it up 😄